packages/@react-types/illustrated-message/src/index.d.ts

```diff
@@ -11,11 +11,11 @@
  */
 
 import {DOMProps, StyleProps} from '@react-types/shared';
-import {ReactElement, ReactNode} from 'react';
+import {ReactElement} from 'react';
+import {Slots} from '@react-types/layout';
 
 export interface SpectrumIllustratedMessageProps extends DOMProps, StyleProps {
-  heading?: string,
-  description?: ReactNode,
-  illustration?: ReactElement,
-  ariaLevel?: number
+  ariaLevel?: number,
+  children: [ReactElement],
+  slots?: Slots
```

Comment:
We shouldn't allow overriding the `IllustratedMessage` slots via a prop I don't think.

Reply:
@snowystinger will address this in his slots followup work per a chat you two had.